### ISSUE TITLE
Unexpected results in the multi-domain DST experiment

### ISSUE THREAD

**Issue by ha-lins** [jasonwu0731/trade-dst#12]
When I run the multi-domain DST experiment , the result is as follows:
![image](https://user-images.githubusercontent.com/22448743/66249570-08194680-e768-11e9-8d05-83c67ba51f15.png)

As you can see, the joint and turn acc is quite low although the loss is reasonable (i.g. 0.5). Is it a normal state? Could you help me with the unexpected results? Besides, when I run the unseen-domain DST experiment, it works well. 

![捕获](https://user-images.githubusercontent.com/22448743/66249642-524ef780-e769-11e9-9cb8-69643ab93093.PNG)

**Comment by jasonwu0731** [comment]
Did you modify any part yourself? Cause this never happens when I run the code. Can u clone the repo and try again?

**Comment by ha-lins** [comment]
Thanks! I clone and try again and then it works well. Maybe it is caused by what I modified. 

What I want to ask as well is that whether the first row in Table 3 is **_the training acc_** of the Base Model on four domains? 
When I run the code,  the fine-tune results on the new domain is about 2-points lower than the reported results, on the condition that it achieves the same training acc.

**Comment by jasonwu0731** [comment]
all numbers reported are test accuracy.
